server/workflows/provision_device.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "provision_device",
+    "description": "Provision device.",
+    "version": 3,
+    "tasks": [
+                    {
+                        "name": "create_device_inventory",
+                        "taskReferenceName": "create_device_inventory",
+                        "inputParameters": {
+                            "http_request": {
+                                "uri": "http://mender-inventory:8080/api/internal/v1/inventory/devices",
+                                "method": "POST",
+                                "contentType": "application/json",
+                                "body": "${workflow.input.device}",
+                                "headers": {
+                                    "X-MEN-RequestID": "${workflow.input.request_id}",
+                                    "Authorization": "${workflow.input.authorization}"
+                                }
+                            }
+                        },
+                        "type": "HTTP"
+                    }
```

Comment:
Why emit event is a separate workflow instead being part of provision_device workflow?

Reply:
Is client supposed to emit event or start `provision_device` workflow?

Reply:
This flow was in the acceptance criteria (or did I misunderstood something?)
 
Acceptance critera:
- Create event that triggers an open-source workflow that:
- Creates a new device via the inventory API

Reply:
poorly expressed acceptance :/

Reply:
it would be less complicated to emit trigger as part of that provisioning (1 workflow + 1 event handler less needed)

also difference in behaviour is that provisioning job have to complete with success before trigger is emitted

Reply:
if the event should be a part of provision_device workflow, we don't need the event handler (at least right now)

Reply:
@maciejmrowiec I've updated the conductor flow according to your comments